File: Baekjoon_Online_Judge/NO_02752/2752.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# ---------------------------- #
# | Created by happiness96   | #
# | Year 2020                | #
# | Month 08                 | #
# | Day 02                   | #
# | 2752 세수정렬              | #
# ---------------------------- #


if __name__ == "__main__":
    l = list(map(int, r_input().split()))

    print(*sorted(l))

File: Baekjoon_Online_Judge/NO_02752/2752.cpp
#include <iostream>
#include <algorithm>
using namespace std;
/*
 ---------------------------- 
 | Created by happiness96   | 
 | Year 2020                | 
 | Month 08                 | 
 | Day 02                   | 
 | 2752 세수정렬              | 
 ---------------------------- 
*/

int main(){
    int numbers[3];

    for(int i = 0; i < 3; i ++) cin >> numbers[i];

    sort(numbers, numbers + 3);

    for(int i = 0; i < 3; i ++) cout << numbers[i] << " ";
    
    return 0;
}
